### DISCUSSION TITLE
Enable sampling from satellite

### DISCUSSION THREAD

**Discussion by Arunkumarlnr** [apache/skywalking#9149] [Q&A] [answered]
Hi team,
Reading the documentation of skywalking satellite I found it provides preprocessing functions to ensure accuracy of the metrics, such as sampling. But I could not find how to add the same. Can you help me with this?

**Comment by wu-sheng** [comment, marked as answer]
No, there is no sampling or pre-calculation mechanism. The part doesn't have enough contributions from the community. No one is leading in that direction.
